In [18]:
import numpy as np
import pandas as pd
import seaborn as sn
import matplotlib as plt



# Data frame

In [20]:
df = pd.read_csv("training.csv")
df.head()

ID  Age  Gender  Marital status  Highest Qualification  Household status  \
0  5086   98       2               3                      1                 1   
1  1258   40       1               1                      3                 5   
2   331   18       2               4                      6                 3   
3  6729   50       1               1                      3                 1   
4  8671   34       1               1                      1                 1   

   Land ownership  Mobile phone users  Salary/wage earner?  \
0               2                   2                    0   
1               1                   1                    1   
2               2                   1                    0   
3               1                   1                    0   
4               2                   1                    0   

   Selling products/goods  ...  Frequency of Mobile money bill pays  \
0                       0  ...                                   -1   
1                       0  ...                                    4   
2                       0  ...                                   -1   
3                       0  ...                                   -1   
4                       1  ...                                   -1   

   Literacy in Kiswhahili  Literacy in English  Latitude  Longitude  \
0                       4                    4 -4.460442  29.811396   
1                       1                    4 -6.176438  39.244871   
2                       1                    1 -6.825702  37.652798   
3                       1                    4 -3.372049  35.808307   
4                       1                    4 -7.179645  31.039095   

   mobile_money  savings  borrowing  insurance  mobile_money_classification  
0             0        0          0          0                            0  
1             1        1          1          0                            3  
2             1        0          0          0                            2  
3             1        0          1          0                            3  
4             1        1          0          1                            3  

[5 rows x 37 columns]

 
 
 
 # Finding if there is any missing values from the data-frame.

In [9]:
display(df.isnull().sum())

ID                                                    0
Age                                                   0
Gender                                                0
Marital status                                        0
Highest Qualification                                 0
Household status                                      0
Land ownership                                        0
Mobile phone users                                    0
Salary/wage earner?                                   0
Selling products/goods                                0
Providing services                                    0
Piece jobs                                            0
Rental income                                         0
Saving/investments                                    0
Pensions                                              0
Government                                            0
Money given by someone                                0
No income                                       

* There are no missing values

# Checking number of males and females

In [8]:
display(df["Gender"].value_counts())


2    3972
1    3122
Name: Gender, dtype: int64

* Males = 3122 and Females = 3972






# Age-data
  ## _The youngest and the oldest person_

In [11]:
print(df['Age'].min())
print(df['Age'].max())

16
100


* Yougest person(s) = 16, Oldest = 100 


 ## Grouping by age-interval

In [12]:
bins = pd.cut(df['Age'] ,[15,24,35,44,55,64,100]).value_counts()
bins

(24, 35]     2249
(15, 24]     1535
(35, 44]     1145
(44, 55]     1009
(64, 100]     638
(55, 64]      518
Name: Age, dtype: int64

 #### Age groups
 * 16 to 24 = 1535
 * 25 to 34 = 2249
 * 35 to 45 = 1145
 * 55 to 64 = 518
 * 65 to 100 = 638

# People who use mobile money

In [14]:
display(df["mobile_money"].value_counts())



1    3930
0    3164
Name: mobile_money, dtype: int64

   * number of people who use mobile money = 3920
   * number of people who dont use mobile money  = 3164


In [ ]:
display(nobel.loc[nobel.prize_share != '1/1'])

In [20]:
git status

SyntaxError: invalid syntax (<ipython-input-20-d93c8dd246e3>, line 1)